phpBB/includes/functions.php

```diff
@@ -3754,41 +3754,6 @@ function obtain_users_online_string($online_users, $item_id = 0, $item = 'forum'
 }
 
 /**
-* Get option bitfield from custom data
-*
-* @param int	$bit		The bit/value to get
-* @param int	$data		Current bitfield to check
-* @return bool	Returns true if value of constant is set in bitfield, else false
-*/
-function phpbb_optionget($bit, $data)
-{
-	return ($data & 1 << (int) $bit) ? true : false;
-}
-
-/**
-* Set option bitfield
-*
-* @param int	$bit		The bit/value to set/unset
-* @param bool	$set		True if option should be set, false if option should be unset.
-* @param int	$data		Current bitfield to change
-*
-* @return int	The new bitfield
-*/
-function phpbb_optionset($bit, $set, $data)
-{
-	if ($set && !($data & 1 << $bit))
-	{
-		$data += 1 << $bit;
-	}
-	else if (!$set && ($data & 1 << $bit))
-	{
-		$data -= 1 << $bit;
-	}
-
-	return $data;
-}
-
-/**
```

Comment:
Sorry Nicofuma, I didn't realize that functions_compatibility is included from common so available everywhere.

However, there might still be a corner case BC issue for extensions: if an extension checks for the existence of one of these functions to include functions.php or not, then the extension may fail now, as the check will indicate that the function exists (from functions_compatibility, included by common), and hence, functions.php will not be included.  If later in the code, another function in functions.php is called (without further checking, as it should already be included), it will not be there.  This is a very extreme corner case, so probably better not doing anything, but just wanted to point it.